#.hlint.yaml#

```diff
@@ -0,0 +1,78 @@
+# HLint configuration file
```

Comment:
Mistake? Feel free to add a gitignore pattern that catches this file.

Reply:
Oh dear. Will fix.